### PR TITLE
BC001 TR01

### PR DESCRIPTION
**Pull request by BenzclyZhang**

### CODE REVIEW

File: tests/TR01/Dataset/Reference_file/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+|------------------------------|-------------------------------------------|
+| Number of alignment(s)       | 11                                        |
+| Vertical Measurement         | Track axis (center line)                  |
+| Properties of segments       | no                                        |
+| Horizontal layout            | Line, Circular Arc, Clothoid              |
+| Vertical layout              | Constant Gradient, Circular Arc           |
+| Cant layout                  | Constant Cant, Linear Transition          |
+| Broken chainage              | No                                        |
+| IFC reference file available | **WIP**                                   |
+
```
Comment (pjanck) [inline tests/TR01/Dataset/Reference_file/README.md:32]: I believe that there is a reference file now.

File: tests/TR01/Dataset/Reference_file/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+## Model Dataset
+This dataset has 11 alignments, which in total are around 18 km long.
+
+| Filename (format)         | Description                                                        |
+|---------------------------|--------------------------------------------------------------------|
+| [BC001_Alignment.xml](./BC001_Alignment.xml)    |    Data containing track alignments in LandXML format                                   |
+| [BC001_Alignment.dwg](./BC001_Alignment.dwg)     |    Data containing track alignments in DWG format                                 |
+| BC001_Alignment(Reference).ifc    |    Data containing track alignments in IFC 4.3                                  |
```
Comment (pjanck) [inline tests/TR01/Dataset/Reference_file/README.md:40]: Is this still up-to-date?

**Review by pjanck — CHANGES_REQUESTED** [review]
Initial comments.

File: tests/TR01/Dataset/Reference_file/README.md (hunk 2 above)
Comment (BenzclyZhang) [reply tests/TR01/Dataset/Reference_file/README.md]: Old file. Removed.

File: tests/TR01/README.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+Formal Rules are those contained in the Gerkin documentation provided within the bSI validation system. In particular, the following list of rules will apply
+
+#### IFC standard (schema and specification)
+When validated using the bSI Validation Service, the IFC must pass:
+
+- Syntax & Schema check
+- All following rules:
+  - ED001 - Each `IfcElement` that is decomposed by other `IfcElement` shall not have body representation.
+  - EP001 - An `IfcProduct` that is positioned based on an `IfcPositioningElement` shall be placed relative to the `ObjectPlacement` of the `IfcPositioningElement`.
+  - EP002 - An `IfcElement` that composes another `IfcElement` shall not have `IfcRelPositions` relationship to an `IfcPositioningElement` or have `IfcRelContainedInSpatialStructure` relationship to an IfcSpatialStructureElement.
```
Comment (pjanck) [inline tests/TR01/README.md:91]:
```suggestion
  - EP002 - An `IfcElement` that composes another `IfcElement` shall not have `IfcRelPositions` relationship to an `IfcPositioningElement` or have `IfcRelContainedInSpatialStructure` relationship to an `IfcSpatialStructureElement`.
```

**Pull request merged**